### DISCUSSION TITLE
Table Plugin Hide Column

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#53691] [Plugins]
Greetings,

I am using the table plugin and I want to hide a column that I want to use for Data Links. Other than using a very small column width how can I do this?

Using Organize transformation to hide it make it unavailable to be used in
```
${__data.fields[1]}
```

What would be the recommended way of doing this or is this a bug in that plugin or in transformation?

Gracias!

**Comment by yosiasz** [comment]
this solves it yes!!
https://github.com/grafana/grafana/issues/24092#issuecomment-1156073661

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
